edfio/edf.py

```diff
@@ -322,6 +322,29 @@ def _set_data(self, data: npt.NDArray[np.float64]) -> None:
         )
         self._digital = np.round(data / gain - offset).astype(np.int16)
 
+    def resample(
```

Comment:
maybe we could name this function `set_resampled_data`, as it does not perform any resampling itself?